posthog/api/decide.py

```diff
@@ -141,7 +141,11 @@ def get_decide(request: HttpRequest):
                     ),
                 )
 
-            property_overrides = get_geoip_properties(get_ip_address(request))
+            property_overrides = {}
+
+            if not data.get("geoip_disable"):
```

Comment:
the double negations 🤖

Reply:
bump on testing the input here too (bool vs string args)

Reply:
sadge